imap_upstream.adoc

```diff
@@ -16,13 +16,11 @@ IMAP related configuration options are available:
 
 The available balancing algorithms are as follows:
 
-* ClientIpHash
+* AccountIdHash
```

Comment:
eww formatting...   This part, and just above should probably be redone with the use of bold (for zimbra config vars and perhaps italics (preferred? or underscore?) for AccountIdHash and custom and em dash (consistency please :)).